### PR TITLE
Editing posts

### LINKED ISSUE #430: Editing a post edited previously using steemit.com gives a blank post body
#### Expected behavior

After editing via any Steem client, a post's text, code and formatting should be editable and shown when the edit button is pressed.

#### Actual behavior

The post body content box is empty when attempting to edit.

image

#### How to reproduce

I posted this via busy.org first: https://busy.org/steem/@pfunk/arguments-for-keeping-the-steem-reward-pool-whole

I was not able to use certain formatting features with the busy editor, and also did not like the way Busy formatted spaces between paragraphs. So I edited the post using steemit.com. Attempting to edit it on Busy again reveals the bug.

* Browser: Firefox 52.0.1

### PR DESCRIPTION
**Pull request by Sekhmet**
- [x] Add edit button depending on post format, authorship and date
- [x] Save draft from post and redirect to editor page
- [x] Reload post after editing
- [x] Add editing unavailable message (wrong format, too late)
- [x] Add translations

Closes #430 #457 

https://trello.com/c/6Ie0lCai/125-edit-post

### CODE REVIEW

File: src/components/Story/StoryFull.js [hunk 1]
```diff
@@ -144,6 +156,49 @@ class StoryFull extends React.Component {
...
+        <i className="iconfont icon-flag" />
+        <FormattedMessage id="report_post" defaultMessage="Report post" />
+      </PopoverMenuItem>,
+    ];
+
+    if (ownPost) {
+      const postMetaData = jsonParse(post.json_metadata);
+      let item = null;
+
+      if (postMetaData.format !== 'markdown') {
```
Comment (bonustrack) [inline src/components/Story/StoryFull.js]: Why limiting the edit to markdown?

File: src/components/Story/Story.js [hunk 2]
```diff
@@ -78,6 +90,45 @@ class Story extends React.Component {
...
+          <FormattedMessage id="edit_post_error_time" defaultMessage="Can't edit posts older than a week" />
+        </PopoverMenuItem>);
+      } else {
+        item = (<PopoverMenuItem key="edit">
+          {saving ? <Icon type="loading" /> : <i className="iconfont icon-write" />}
+          <FormattedMessage id="edit_post" defaultMessage="Edit post" />
+        </PopoverMenuItem>);
+      }
+
+      popoverMenu.push(item);
```
Comment (bonustrack) [inline src/components/Story/Story.js]: "Edit post" should be the first element in the menu

**Comment by bonustrack** [comment]
See how edit is done with `diff` on Busy` https://github.com/busyorg/busy/blob/95c6dfee70b9018193660a2207a689243f636459/src/post/Write/EditorActions.js#L12

**Comment by bonustrack** [comment]
Also `Following @hellosteem` when i'm logged with @hellosteem should be removed from the menu.

File: src/components/Story/StoryFull.js (hunk 1 above)
Comment (Sekhmet) [reply src/components/Story/StoryFull.js]: This was handled this way in Busy1.
I will change this.

**Comment by bonustrack** [comment]
- On editor when edit a post, "Reward" and "Like this post" should not be visible (you cant update rewards after posting)
- Editor button should be "Update post" instead of "Post"

**Pull request merged**